### ISSUE TITLE
Update Testing Docs regarding Primer

### ISSUE THREAD

**Issue by clavedeluna** [PyCQA/pylint#7714]
### Bug description

[Primer testing docs](https://pylint.pycqa.org/en/v2.12.2/development_guide/testing.html#primer-tests) mention `batch_one` and `batch_two`:

![Screen Shot 2022-11-05 at 9 27 00 AM](https://user-images.githubusercontent.com/112832187/200119831-46cd1f0e-15a3-4d4e-91e1-8ac0f43863c7.png)

at the very least, the suggested command with batch_two doesn't run any tests.

but I don't find any `batch_two` in the repo. Should this documentation be updated?

### Command used

```shell
n/a
```

### Pylint output

```shell
n/a
```

### Expected behavior

n/a

### Pylint version

```shell
n/a
```

### OS / Environment

n/a

**Comment by jacobtylerwalls** [comment]
Yes, this is very out of date. Not only is there no longer a batch two, we aren't even running batch one anymore on CI. The new primer is in tests/primer/__main__.py

**Comment by mbyrnepr2** [comment]
https://pylint.pycqa.org/en/latest/development_guide/contributor_guide/tests/launching_test.html#primer-tests

**Comment by clavedeluna** [comment]
Ah! No I think I actually made a mistake. The link I provided 
https://pylint.pycqa.org/en/v2.12.2/development_guide/testing.html#primer-tests
is clearly for an old version. Google just took me to that and I didn't notice.

Given https://pylint.pycqa.org/en/v2.15.5/development_guide/contributor_guide/tests/launching_test.html exists and has the correct docs, I'll close this issue.

Thanks for clarifying!

**Comment by jacobtylerwalls** [comment]
That documentation is still out of date. It only refers to the old primer.

**Comment by clavedeluna** [comment]
See https://github.com/PyCQA/pylint/pull/7685#issuecomment-1309387417 for how to run primer

**Comment by mbyrnepr2** [comment]
Would it be an idea to add the primer commands to tox and document how to run the primer using tox?

**Comment by Pierre-Sassoulas** [comment]
That could work but primer are slow and tox also make things slower.  (We'd have to check that the primed repoes are not downloaded by tox each time). I guess new contributor would appreciate to have a ``tox -e primer`` command and not having to read the full doc, and maintainer will know the faster command.

Regardless of what we do to make it easier to launch primer, we can close this issue once the primer are documented more clearly in the doc.

**Comment by clavedeluna** [comment]
I would be all for documenting the full commands
```
python tests/primer/__main__.py prepare --clone
python tests/primer/__main__.py run --type=pr
```

while also adding to tox and indicating both can be used.

As a frequent contributor, it took me waaay to long to figure out how to run primer locally...

**Comment by Pierre-Sassoulas** [comment]
Also it should probably be launchable with a single command and no argument (``python tests/primer/__main__.py``).

**Comment by DanielNoord** [comment]
It was a conscious decision to separate the commands to be able to easily reuse currently cloned project.

We could add an `full-run` parser but good documentation would probably be more effective and easier.

**Comment by Pierre-Sassoulas** [comment]
> It was a conscious decision to separate the commands to be able to easily reuse currently cloned project.

Sure and it's a good thing, but we can auto-detect that the repositories were already cloned.
